src/Symfony/Component/DomCrawler/Form.php

```diff
@@ -197,9 +197,21 @@ public function getUri()
     {
         $uri = parent::getUri();
 
-        if (!in_array($this->getMethod(), array('POST', 'PUT', 'DELETE', 'PATCH')) && $queryString = http_build_query($this->getValues(), null, '&')) {
-            $sep = false === strpos($uri, '?') ? '?' : '&';
-            $uri .= $sep.$queryString;
+        if (!in_array($this->getMethod(), array('POST', 'PUT', 'DELETE', 'PATCH'))) {
+            $query = parse_url($uri, PHP_URL_QUERY);
+            $currentParameters = array();
+            if ($query) {
+                foreach (explode('&', $query) as $param) {
```

Comment:
Unfortunately, as soon on PHP.net:

 > Parses str as if it were the query string passed via a URL and **sets variables in the current scope.**

There is no easy (nor quick) way to find out which variables were added in the current scope IRCC